client/my-sites/upgrades/domain-management/style.scss

```diff
@@ -437,9 +436,8 @@
 	}
 
 	strong {
-		background: url('/calypso/images/upgrades/google-apps-logo.png');
-		background-position: left center;
-		background-repeat: no-repeat;
+		//noinspection CssUnknownTarget
```

Comment:
It stops Webstorm from complaining about unknown target. But I guess we're not using anything to supress Webstorm errors and it might be best to remove it actually